pinot-common/src/main/java/org/apache/pinot/common/metrics/BrokerQueryPhase.java

```diff
@@ -30,6 +30,8 @@
   QUERY_EXECUTION,
   QUERY_ROUTING,
   SCATTER_GATHER,
+  SERVER_PROCESSING,
+  NETTY_CONNECTION,
```

Comment:
This I believe is used to measure the time it took for the server response to travel over the wire, arrive at broker and get deserialized. We already emit this from the server but I don't think it is at the table level
See the following code in InstanceRequestHandler

`_serverMetrics.addTimedValue(ServerTimer.NETTY_CONNECTION_SEND_RESPONSE_LATENCY, sendResponseLatencyMs, TimeUnit.MILLISECONDS);`

Reply:
The metric you mentioned doesn't exclude the time spent in processing the query in server, which isn't the purpose of this PR.

Reply:
It excludes the query processing time. See code in Instance Request Handler

Reply:
Refactored the code to put `NETTY_CONNECTION_SEND_REQUEST_LATENCY` to broker timer metrics.